dataverse-persistence/src/main/resources/db/migration/V12__addedUserPreferredLanguage.sql

```diff
@@ -0,0 +1,6 @@
+ALTER TABLE authenticateduser
+    add column if not exists notificationslanguage varchar(2);
```

Comment:
I would loosen up type of column to `text`. `varchar(2)` is pretty restrictive and we would be unable to store something like `en_US` or similar